command.go

```diff
@@ -850,42 +852,64 @@ func (c *Command) HasSubCommands() bool {
 	return len(c.commands) > 0
 }
 
+// Determine if a command is available as a non-help command (this includes all
```

Comment:
I'm fine doing it, I appreciate the feedback. Normally I do start with the name of the function for godocs, however recently I heard somewhere that was no longer a requirement (but I'm actually fond of it, so I don't mind doing it anyway).